07-plotting-solar-data/displaying_data.ipynb

```diff
@@ -52,12 +63,21 @@
      ]
     },
     {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Map.peek() shows a basic display of the Map's data with a title, axis labels and a colour bar. It also returns a matplotlib figure object."
+     ]
+    },
+    {
      "cell_type": "code",
      "collapsed": false,
      "input": [
       "# Import Map and make a Map object to display\n",
       "from sunpy.map import Map\n",
```

Comment:
This idiom should not be encouraged. My personal preference is for `import sunpy.map`.